<launch>
    <node pkg="joy" type="joy_node" name="joy_node" ></node>
    <node pkg="wheelleg_quad" type="chassisController" name="chassisController" ></node>
    <node pkg="wheelleg_quad" type="motorController" name="motorController" ></node>
</launch>